vcpkg.json

```diff
@@ -23,10 +23,7 @@
     ],
     "features": {
         "tests": {
-            "description": "Build with tests",
-            "dependencies": [
-                "gtest"
-            ]
+            "description": "Build with tests"
```

Comment:
I don't remember why we did this. Can you add this back?

Reply:
gtest binaries are incompatible w/IntelLLVM and cause SEH exceptions. we will need a separate toolchain file. to be addressed in separate PR

Reply:
Can you not add this to the main vcpkg and only do it locally while we figure out a solution. This change will break the CI and all the other backends.